src/axolotl/utils/config/models/input/v0_4_1/__init__.py

```diff
@@ -221,12 +221,17 @@ class ModelInputConfig(BaseModel):
     tokenizer_type: Optional[str] = Field(
         default=None, metadata={"help": "transformers tokenizer class"}
     )
-    model_type: Optional[str] = Field(default=None)
+    model_type: Optional[str] = Field(default=None, alias="type_of_model")
     model_revision: Optional[str] = None
```

Comment:
```suggestion
    model_revision: Optional[str] = Field(default=None, alias="revision_of_model")
```
should probably update this as well.